src/main/flight/imu.c

```diff
@@ -423,12 +424,19 @@ static void imuCalculateEstimatedAttitude(void)
     imuCalculateAcceleration(deltaT); // rotate acc vector into earth frame
 }
 
-void imuUpdate(rollAndPitchTrims_t *accelerometerTrims)
+void imuUpdateAccelerometer(rollAndPitchTrims_t *accelerometerTrims)
 {
-    gyroUpdate();
-
     if (sensors(SENSOR_ACC)) {
         updateAccelerationReadings(accelerometerTrims);
+        isAccelUpdatedAtLeastOnce = true;
+    }
+}
+
+void imuUpdate(void)
```

Comment:
I would rather separate gyro and acc update like
void imuUpdateAcc
and
void imuUpdateGyro